docs/apache-airflow/project.rst

```diff
@@ -87,6 +87,7 @@ Committers
 - Tomasz Urbaszek (@turbaszek)
 - Tzu-ping Chung (@uranusjr)
 - Vikram Koka (@vikramkoka)
+- Vincent Beck(@vincbeck)
```

Comment:
```suggestion
- Vincent Beck (@vincbeck)
```

Reply:
Oh god... We have ANOTHER committer list :)

Reply:
@vincbeck - can you please add @mobuchowski and @phanikumv as well while correcting the space :). Pretty please :)

Reply:
And I will correct instructions

Reply:
Ah .. it was already there//

Reply:
> @vincbeck - can you please add @mobuchowski and @phanikumv as well while correcting the space :). Pretty please :)

Will do!

Reply:
:bow: